### ISSUE TITLE
📝 Docs: Website has React key complaints

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [aidenybai/million#869]
When I run the website locally and visit the homepage, I get the dreaded https://reactjs.org/link/warning-keys complaint:

```plaintext
Warning: Each child in a list should have a unique "key" prop.

Check the top-level render call using <...>. See https://reactjs.org/link/warning-keys for more information.
```

...in two components:

https://github.com/aidenybai/million/blob/65ef9aff64597f56feb4401d13519844badc533c/website/components/home/hero.tsx#L155-L156

https://github.com/aidenybai/million/blob/65ef9aff64597f56feb4401d13519844badc533c/website/components/home/showcase.tsx#L72-L77

There's something amusing about a performance library triggering the React key complaint that's commonly associated with performance (albeit on a relatively static docs site).

Is there a reason React ESLint plugins to catch this aren't enabled on the website?

**Comment by tobySolutions** [comment]
Thank you so much for this @JoshuaKGoldberg!
